org.openhab.binding.zigbee.firmware/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.zigbee.reactor</artifactId>
+    <version>3.3.0-SNAPSHOT</version>
```

Comment:
This seems to break our release build (https://ci.openhab.org/view/Sandbox/job/sandbox-openhab3-release/) since October 11 - it should clearly be `3.4.0-SNAPSHOT`.